.gofmt.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+fmt_errors=$(gofmt -d .)
```

Comment:
Not entirely sure why. I think we've copied that from some other repo. The "-s" part is more important since that actually triggers a larger set of analysis.